src/graphql/outlier.rs

```diff
@@ -342,6 +342,44 @@ impl From<PreserveOutliersInput> for review_database::OutlierInfoKey {
     }
 }
 
+#[derive(Debug, Clone, Serialize)]
+pub struct PreserveOutlierOutput {
+    id: i64,
+    model_id: i32,
+    timestamp: i64,
+    source: String,
+}
+
+impl From<PreserveOutliersInput> for PreserveOutlierOutput {
+    fn from(input: PreserveOutliersInput) -> Self {
+        Self {
+            model_id: input.model_id,
+            timestamp: input.timestamp,
+            id: input.id,
+            source: input.source,
+        }
+    }
+}
+
+#[Object]
+impl PreserveOutlierOutput {
+    async fn id(&self) -> ID {
+        ID(self.id.to_string())
+    }
+
+    async fn model_id(&self) -> StringNumber<i32> {
+        StringNumber(self.model_id)
+    }
```

Comment:
What about using `i32`?

Reply:
Since I used StringNumber mechanically, I made a mistake to apply it to the i32.
I'll make a correction.